### PR TITLE
Reworded the introduction

### PR DESCRIPTION
**Pull request by jakozaur**
More readable.

### CODE REVIEW

File: src/introduction.tex [hunk 1]
```diff
@@ -46,15 +46,17 @@
 
 \subsubsection{Statement of the problem}
 
-Nowadays the industry increasingly relies on cloud technologies. More and more companies start or move their products to cloud environment.
-It has been observed that gradually institutions introduce dedicated teams for managing the costs of cloud operations or use third-parties for that purpose. Such solutions cause great savings for everyone.
-Unfortunately existing toolkit is either very primitive or targets billion dollar companies. Currently neither Amazon nor any third-party offer a tool that would allow flexible, cheap and quick insight into the operation costs for small or middle-sized organizations.
+The software industry increasing adopt public cloud as the primary method of running their infrastructure.
```
Comment (mbalc) [inline src/introduction.tex]: should this be `increasingly adopts`? I'm not sure but in this form the grammar doesn't make sense to me

File: src/introduction.tex [hunk 2]
```diff
@@ -46,15 +46,17 @@
...
+
+As the cost of cloud increases, the companies invest in dedicated teams or tools to improve visibility and drive savings. Most of the current solution aims to reduce the cost of production workloads. However, systems for research and development have very different patterns, poorly addressed by existing tools.
```
Comment (mbalc) [inline src/introduction.tex]: imo `most of the current solutions aim`*

Comment (mbalc) [inline src/introduction.tex]: very different -> distinct, might be a better wording but I'm not sure if this fits into what was meant to convey here, mark as resolved if it doesn't (optionally, find a way to express this divergence in some better way)

Comment (mbalc) [reply src/introduction.tex]: cause I'm not sure what do we specifically mean here with `research`

File: src/introduction.tex [hunk 3]
```diff
@@ -46,15 +46,17 @@
...
+The software industry increasing adopt public cloud as the primary method of running their infrastructure.
+
+As the cost of cloud increases, the companies invest in dedicated teams or tools to improve visibility and drive savings. Most of the current solution aims to reduce the cost of production workloads. However, systems for research and development have very different patterns, poorly addressed by existing tools. 
 
 \subsubsection{The solution}
 
-The Omigost is a system that offers reliable solution adressing that issue. We focus on one of the biggest cloud platforms - Amazon Web Services, which is maintained by Amazon.
-Whole system intends not only to provide easy cost insight, but also help with basic cost optimization.
-Despite collecting and storing AWS data the system will offer users following core features:
+The Omigost is a system that addresses the research and development cost saving use case. We focus on one of the biggest cloud platform - Amazon Web Services, which is managed by Amazon.
```
Comment (mbalc) [inline src/introduction.tex]: one of the biggest cloud platform**s**

**Review by mbalc — CHANGES_REQUESTED** [review]

File: src/introduction.tex (hunk 3 above)
Comment (jakozaur) [reply src/introduction.tex]: Disagree. It is a single platform. The plural would be `the biggest cloud platforms are Azure, AWS, GCP`.

Comment (mbalc) [reply src/introduction.tex]: yeah, it is a single platform, thus we have the word `one` there - but, it's one *from among a range* of many other *biggest cloud platforms*, that's what the phrase is about, the plural here doesn't indicate we consider a multitude of them, but that the AWS is not the only big cloud provider that is out there

Comment (mbalc) [reply src/introduction.tex]: some backing:
https://blog.harwardcommunications.com/2012/05/25/one-of-each-of-any-of-plus-plural/

File: src/introduction.tex (hunk 1 above)
Comment (mbalc) [reply src/introduction.tex]: this is closer now, but still doesn't compile gramatically for me - we have `software industry` which has a role of an actor, a noun - and just after that the `increasing adoption of cloud resources` that also has a role of a noun - there's no correlation between those, we probably need a verb such as `is`, ie `software industry is increasing adoption`

Comment (mbalc) [reply src/introduction.tex]: so - this might either just be missing that `is` there, or you want to use some other phrase  here

**Pull request merged**